src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.ts

```diff
@@ -27,6 +42,75 @@ export class LogsComponent implements OnInit, OnDestroy {
   getInfo() {
     this.logsService.getLogs().subscribe((data: any) => {
       this.contentData = data;
+      this.filterLogs();
+    });
+  }
+
+  transformDateType(): any {
+    let year_month_day: string;
+    let s_time: number, e_time: number;
+
+    if (this.date) {
+      year_month_day = this.date.toISOString().slice(0, 10);
+    } else {
+      year_month_day = null;
+    }
+
+    const s_hour = this.startTime.getUTCHours();
```

Comment:
In general, we are using camel case in frontend code as recommended by Style Guide:

https://angular.io/guide/styleguide#style-03-04